api-client/client.ts

```diff
@@ -181,7 +181,7 @@ const parseResponseBody = async (
 ) => {
   try {
     return await responseSchema.parseAsync(json)
-  } catch {
+  } catch (e) {
```

Comment:
Do you want to print this error?

Reply:
I can make a PR in the shared repo for this.